Convert database from Table Storage to Cosmos DB

duytan8484 opened the pull request.

duytan8484 committed:
Convert database from Table Storage to Cosmos DB

duytan8484 committed:
Support pagination by using continuation token

hai-tran commented on DustStream/Services/CosmosDbHelper.cs:
```diff
@@ -0,0 +1,134 @@
+
+                // Replace the item
+                existedItem = await this.CosmosDbContainer.ReplaceItemAsync<T>(item, keyString, new PartitionKey(partitionString));
+                Console.WriteLine("Updated item [{0},{1}].\n \tBody is now: {2}\n", partitionString, keyString, existedItem.Resource);
```
Should we remove these debug lines?

duytan8484 committed:
Use mat-paginator instead of ngb-pagination
Refactorring based on pull request comments

duytan8484 committed:
Handle project view when there is new added item event

duytan8484 committed:
Added tools, scripts, and brief instruction regarding migrating database

duytan8484 committed:
- Combine tables' name and individual partition key to be only one partition key of single container
- Update scripts to migrate data from table storage to single container cosmosdb
